.github/workflows/nightly.yml

```diff
@@ -58,3 +58,65 @@ jobs:
     - name: Fuzz long test
       working-directory: ${{github.workspace}}/build
       run: ctest -C ${{matrix.build_type}} --output-on-failure -L "fuzz-long"
+
+  adapter-build:
+    name: Build - Adapters on Ubuntu
+    strategy:
+      matrix:
+        adapter: [L0]
+        build_type: [Debug, Release]
+        compiler: [{c: gcc, cxx: g++}, {c: clang, cxx: clang++}]
+
+    runs-on: self-hosted
```

Comment:
when making the PR to the upstream, please rename it to e.g. `devcloud` or `devcloud-pvc` perhaps